src/hub/ReclaimQueue.sol

```diff
@@ -255,7 +257,7 @@ contract ReclaimQueue is IReclaimQueue, Pausable, Ownable2StepUpgradeable, UUPSU
 
     // run actual claim logic
     ClaimResult memory res = _claim($, receiver, vault);
-    require(res.totalAssetsClaimed > 0, IReclaimQueue__NothingToClaim());
+    require(res.reqIdFrom < res.reqIdTo, IReclaimQueue__NothingToClaim());
```

Comment:
If there is already zero amount of shares or assets, users should be able to claim them.